CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.28.1]
+### Changed
+- Fix master build  all validate-source-maps-beta steps in config.yml into the respective .sh file ([#18427](https://github.com/MetaMask/metamask-extension/pull/18427))
```

Comment:
Nit: The audience for this changelog is external stakeholders, so we should avoid referencing specific scripts and using internal terminology.
```suggestion
- Fix release automation ([#18427](https://github.com/MetaMask/metamask-extension/pull/18427))
```